ms/MeasurementSets/MSIter.h

```diff
@@ -60,11 +60,11 @@ class MSInterval : public BaseCompare
   explicit MSInterval(Double interval) : interval_p(interval), offset_p(0) {}
     virtual ~MSInterval() {}
     virtual int comp(const void * obj1, const void * obj2) const;
-    Double getOffset() {return offset_p;}
-    void setOffset(Double offset) {offset_p=offset;}
-    Double getInterval() {return interval_p;}
+    Double getOffset() const {return offset_p;}
+    virtual void setOffset(Double offset) {offset_p=offset;}
+    Double getInterval() const {return interval_p;}
     void setInterval(Double interval) {interval_p=interval;}
-private:
+protected:

```

Comment:
I'd agree completely were this a question of making the instance variables public.  My intent here was to make them directly accessible in a derived class that implements a smarter comp() function.  The comp() function is called an enormous number times (~nrow), and I'd hoped to avoid the function calls and obj copies of the getter methods (though perhaps the compiler can be relied upon to inline them).   
Nonetheless, I will make local (alas, redundant) instances of these variables in my derived class.